views/restaurants/index.ejs

```diff
@@ -1,37 +1,21 @@
 <%- contentFor('title') %>EcoFoodie
 <%- contentFor('body') %>
 <link rel="stylesheet" type="text/css" href="index.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
```

Comment:
If we're using FontAwesome icons on this page, it's likely we'll need/want to use them on other pages as well. I would move this stylesheet into the main layout.ejs file so FontAwesome icon styles can be used anywhere.